src/Reader/readpar.f90

```diff
@@ -2634,13 +2634,12 @@ SUBROUTINE readpar_output(EQN,DISC,IO,MPI,CalledFromStructCode)
       Refinement = 0
       pickdt = 0.1
       pickDtType = 1
-      nRecordPoints = 0
       energy_output_on = 0
       pickdt_energy = 1.0
       OutputRegionBounds(:) = 0.0
-!      RFileName = 'RecordPoints'
+      RFileName = ''
+      nRecordPoints = -1
```

Comment:
What will happen when we set nRecordPoints = -1?

Reply:
Nothing. It's unused - it's just an easy hack to see whether it was used in the parameters.par - if yes, it issues a warning